pkg/cmd/roachprod-microbench/main.go

```diff
@@ -48,10 +48,35 @@ Typical usage:
 	command.AddCommand(makeCompareCommand())
 	command.AddCommand(makeRunCommand())
 	command.AddCommand(makeExportCommand())
+	command.AddCommand(makeCleanCommand())
 
 	return command
 }
 
+func makeCleanCommand() *cobra.Command {
+	config := defaultCleanConfig()
+	runCmdFunc := func(cmd *cobra.Command, commandLine []string) error {
+		args, _ := splitArgsAtDash(cmd, commandLine)
+
+		config.inputFilePath = args[0]
+		config.outputFilePath = args[1]
+		c, err := newClean(config)
+		if err != nil {
+			return err
+		}
+
+		return c.cleanBenchmarkOutputLog()
+	}
+	command := &cobra.Command{
+		Use:   "clean <inputFilePath> <outputFilePath>",
+		Short: "remove noisy logs from the benchmark output and dump it to a file",
+		Long:  `remove noisy logs from the benchmark output and dump it to a file`,
```

Comment:
If I understand things right, this is kind of summarizing benchmark results and writing the summary to a file. Using `remove` here sounds like it's performing a destructive operation, which is not the case.

It's also unclear what "it" refers to in this sentence.

Reply:
Understood, will update the documentation.